classes/order/Order.php

```diff
@@ -225,6 +230,7 @@ class OrderCore extends ObjectModel
             'reference' => ['type' => self::TYPE_STRING],
             'date_add' => ['type' => self::TYPE_DATE, 'validate' => 'isDate'],
             'date_upd' => ['type' => self::TYPE_DATE, 'validate' => 'isDate'],
+            'note' => ['type' => self::TYPE_STRING, 'validate' => 'isString'],
```

Comment:
Since this field is a `text` it can be injected with a lot of content, maybe we should have a stricter validation method here to avoid injecting HTML or javascript code

@PierreRambaud what do you think?